### ISSUE TITLE
Can't chage single date when range is true

### ISSUE THREAD

**Issue by andreazorzi** [t1m0n/air-datepicker#613]
Hi,
I'm coding an application that implements the datepicker with range: true.
But if I select a date range and then deselect a date to change the range duration, when I click the new date the datepicker reselect the old selected date.
I also try to remove all settings when I initialize the plugin and leave only the date range, but the bug persist.
I am currently use the version 3.5.1. and the code I use is the following

```js
delivery_date = new AirDatepicker('#delivery-date', {
    range: true,
});
```

https://github.com/t1m0n/air-datepicker/assets/15615862/1a2b0b7c-cb6f-453a-bd50-945a1b4d3bbd

**Comment by t1m0n** [comment]
Hello! Thank you for the bug report, I'll check it

**Comment by t1m0n** [comment]
fixed in 3.5.2

**Comment by t1m0n** [comment]
Not fixed yet, found another problem(

**Comment by t1m0n** [comment]
fixed in v3.5.3 )
